### PR TITLE
fix hover effect of back to link

### LINKED ISSUE #8635: Back to bookings link disappears on hover
### Issue Summary

_Back to bookings_ link when previewing a booking has a hover state that blends in the background
Screen Recording 2023-05-02 at 11 20 14 PM

### Steps to Reproduce

1. Make an event type
2. Preview
3. Finish the flow then try to hover over this link (top left)

### Technical details
- Firefox on Mac, not sure if user theme (dark) has anything to do with this

### PR DESCRIPTION
**Pull request by sumitshinde-84**
## What does this PR do?

Fixes  #8635

 Loom Video: https://www.loom.com/share/f2696aa2b2314a3f90f0e42fc34c687b

**Environment**:  Production

### CODE REVIEW

**Review by sean-brydon — APPROVED** [review]
LGTM - thanks for fixing this

**Comment by sumitshinde-84** [comment]
Great to hear that! I'm always eager to tackle more issues and improve our project

File: packages/ui/components/form/timezone-select/TimezoneSelect.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export function TimezoneSelect({
         menu: (state) =>
           classNames(
             "rounded-md bg-default text-sm leading-4 text-default mt-1 border border-subtle",
+            state.selectProps.menuIsOpen && "shadow-dropdown", // Add box-shadow when menu is open
```
Comment (Udit-takkar) [inline packages/ui/components/form/timezone-select/TimezoneSelect.tsx:90]: This is not related to this issue

File: apps/web/pages/booking/[uid].tsx [hunk 2]
```diff
@@ -309,8 +309,8 @@ export default function Success(props: SuccessProps) {
         <div className="mt-2 ml-4 -mb-4">
           <Link
             href={allRemainingBookings ? "/bookings/recurring" : "/bookings/upcoming"}
-            className="hover:bg-subtle text-subtle dark:hover:text-inverted hover:text-default mt-2 inline-flex px-1 py-2 text-sm dark:hover:bg-transparent">
-            <ChevronLeft className="h-5 w-5" /> {t("back_to_bookings")}
+            className="hover:bg-subtle text-subtle  hover:text-default mt-2 inline-flex px-1 py-2 text-sm dark:hover:bg-transparent">
+            <ChevronLeft className="h-5 w-5" /> {t("back_to_bookings")}
```
Comment (Udit-takkar) [inline apps/web/pages/booking/[uid].tsx:313]: lint is failing here. install prettier in turn on format on save in vs code

**Review by Udit-takkar — CHANGES_REQUESTED** [review]
left few comments that needs to be addressed

File: packages/ui/components/form/timezone-select/TimezoneSelect.tsx (hunk 1 above)
Comment (sumitshinde-84) [reply packages/ui/components/form/timezone-select/TimezoneSelect.tsx:90]: "I have submitted a pull request yesterday for this issue #8582, which has not yet been merged. That is why these changes are still in this pull request

**Review by Udit-takkar — COMMENTED** [review]
hey @sumitshinde-84 sorry we would have to close this PR because @roae reviewed and merged another PR https://github.com/calcom/cal.com/pull/8650 to fix the issue
